src/annotator/anchoring/html.ts

```diff
@@ -63,50 +66,45 @@ export function anchor(root, selectors, options = {}) {
 
   // From a default of failure, we build up catch clauses to try selectors in
   // order, from simple to complex.
-  /** @type {Promise<Range>} */
-  let promise = Promise.reject('unable to anchor');
+  let promise: Promise<Range> = Promise.reject('unable to anchor');
 
   if (range) {
     // Const binding assures TS that it won't be re-assigned when callback runs.
     const range_ = range;
     promise = promise.catch(() => {
-      let anchor = RangeAnchor.fromSelector(root, range_);
+      const anchor = RangeAnchor.fromSelector(root, range_);
```

Comment:
Heh, I always end up with a bunch of these `let`->`const` updates when I do these conversions, too...

Reply:
Yep. I don't know why the linter complains about these when the file is TS, but lets it pass when it's JS 🤔